packages/flutter/lib/src/cupertino/scrollbar.dart

```diff
@@ -183,6 +189,25 @@ class _CupertinoScrollbarState extends State<CupertinoScrollbar> with TickerProv
         ..color = CupertinoDynamicColor.resolve(_kScrollbarColor, context)
         ..padding = MediaQuery.of(context).padding;
     }
+    WidgetsBinding.instance.addPostFrameCallback((duration) {
+      if (widget.displayAlways) {
+        assert(widget.controller != null);
+        widget.controller.position.didUpdateScrollPositionBy(0);
+      }
+    });
+  }
+
+  @override
+  void didUpdateWidget(CupertinoScrollbar oldWidget) {
+    super.didUpdateWidget(oldWidget);
+    if (widget.displayAlways != oldWidget.displayAlways) {
+      assert(widget.controller != null);
+      if (widget.displayAlways == false) {
+        _fadeoutAnimationController.reverse();
+      } else {
+        widget.controller.position.didUpdateScrollPositionBy(0);
```

Comment:
Also, I think you'll need to change the logic for these fadein/out animations here so that they only happen when the scrollbar is not scrolling.

Reply:
One more thing.  If we keep this `didUpdateScrollPositionBy`, then you may need to prevent `_fadeoutAnimationController.forward();` in `_handleScrollNotification` below in the case the widget has just been rendered and displayAlways is false.